xbmc/Application.cpp

```diff
@@ -4466,7 +4468,7 @@ float CApplication::GetVolumeRatio() const
 void CApplication::VolumeChanged()
 {
   CVariant data(CVariant::VariantTypeObject);
-  data["volume"] = GetVolumePercent();
+  data["volume"] = std::lroundf(GetVolumePercent());
```

Comment:
You may still need to static cast to int here.

Reply:
@phunkyfish Yes, I added `static_cast<int>(...)` to cast `long int` to `int`.